### PR TITLE
data: add support for Logitech M500s mouse

### PR DESCRIPTION
**Pull request by FoxVK**
Support for Logitech M500s mouse.

I am working on svg for piper but this is not my cup of tea...

### CODE REVIEW

File: data/devices/logitech-M500s.device [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Logitech M570
```
Comment (FFY00) [inline data/devices/logitech-M500s.device:1]: Can you fix this comment? Thanks!

**Review by FFY00 — CHANGES_REQUESTED** [review]

**Comment by FFY00** [comment]
Also, you'll need to add this file to `meson.build`, that's why the CI is failing.

File: data/devices/logitech-M500s.device (hunk 1 above)
Comment (FoxVK) [reply data/devices/logitech-M500s.device]: Uhh sorry.. I am blind...

**Review by FFY00 — APPROVED** [review]

**Comment by FFY00** [comment]
The CI failing on arch is unrelated and expected.

**Pull request merged**